deploy/deploy_testnet.py

```diff
@@ -54,6 +54,10 @@ def main(ctx, **kwargs):
     help='Auction funds will be sent to this wallet address.'
 )
 @click.option(
+    '--whitelister',
```

Comment:
maybe add `required=True` or a default value, otherwise there'll be `KeyError` if someone forgets to set this option.